### ISSUE TITLE
NTR: Renal anemia

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10616]
**Preferred term label:**
Renal anemia

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://doi.org/10.1016/j.semnephrol.2006.06.001

Anemia due to renal disease. Anemia is a common complication of chronic kidney disease. Although mechanisms involved in the pathogenesis of renal anemia include chronic inflammation, iron deficiency, and shortened half-life of erythrocytes, the primary cause is deficiency of erythropoietin (EPO). Serum EPO levels in patients with chronic kidney disease are usually within the normal range and thus fail to show an appropriate increase with decreasing hemoglobin levels, as found in nonrenal anemias. As with other anemias, the consequences of renal anemia are a moderate decrease in tissue oxygen tensions and counterregulatory mechanisms that maintain total oxygen consumption, including a persistent increase in cardiac output.

**Parent term (use hpo.jax.org/app)**
HP:0001903: Anemia

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 
Although this is a well recognized phenomenon, the HPO dies not encode etiologies. The way to encode this for a patient would be to encode with terms for Anemia and another term for Failure to increased EPO.

These are the EPO terms we have now
[Abnormal circulating erythropoietin concentration HP:0034442 ](https://hpo.jax.org/browse/term/HP:0034442)

What about a new term

Inappropriate normal or low erythropoietin
Def: Erythropoietin is produced and released into the blood by the kidneys in response to hypoxemia, and would normally be expected to increase with decreasing hemoglobin levels. This term applies if the increase fails to occur, as may be seen in conditions including chronic kidney disease.

**Comment by GrieseAG** [comment]
Hi Peter, Sorry for this incorrect assignment of an etiology. Sometimes, it is hard to distinguish. I totally agree with your new term suggestion. Thank you very much.

**Comment by pnrobinson** [comment]
@GrieseAG 

Adding new term: 
- ID: HP:6001016
- label: Inappropriate normal or low erythropoietin
- parents: [Abnormal circulating erythropoietin concentration (HP:0034442)](https://hpo.jax.org/app/browse/term/HP:0034442)
- pmid: PMID:32311841
- def: Erythropoietin is produced and released into the blood by the kidneys in response to hypoxemia, and would normally be expected to increase with decreasing hemoglobin levels. This term applies if the increase fails to occur, as may be seen in conditions including chronic kidney disease.
- comment: 
- orcid: orcid:0000-0002-7371-8158
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10616
- synonyms:
